fuse/new_orleans_pd.py

```diff
@@ -162,21 +150,25 @@ def fuse_events(pprr_ipm, pprr_csd, cprr, uof, award, lprr, sas):
     officer_number_dict = create_officer_number_dict(pprr_ipm)
     cprr = pd.read_csv(deba.data("clean/cprr_new_orleans_pd_1931_2020.csv"))
     actions = pd.read_csv(deba.data("clean/cprr_actions_new_orleans_pd_1931_2020.csv"))
-    uof = pd.read_csv(deba.data("clean/uof_new_orleans_pd_2012_2019.csv"))
+    uof_officers = pd.read_csv(deba.data("clean/uof_officers_new_orleans_pd_2016_2021.csv"))
+    uof_citizens = pd.read_csv(deba.data("clean/uof_citizens_new_orleans_pd_2016_2021.csv"))
+    uof = pd.read_csv(deba.data("clean/uof_new_orleans_pd_2016_2021.csv"))
     post_event = pd.read_csv(deba.data("match/post_event_new_orleans_pd.csv"))
     award = pd.read_csv(deba.data("match/award_new_orleans_pd_2016_2021.csv"))
     lprr = pd.read_csv(deba.data("match/lprr_new_orleans_csc_2000_2016.csv"))
     sas = pd.read_csv(deba.data("match/sas_new_orleans_pd_2017_2021.csv"))
     complaints = fuse_cprr(cprr, actions, officer_number_dict)
-    use_of_force = fuse_use_of_force(uof, officer_number_dict)
-    personnel = fuse_personnel(pprr_ipm, lprr, pprr_csd, sas)
+    personnel = fuse_personnel(pprr_ipm, lprr, pprr_csd, sas, uof_officers)
```

Comment:
Shouldn't you save the `uof_officers` in a file that begin with `fuse/uof_officers_` and end up in an actual table on wrglhub that is called `uof_officers`? Don't get me wrong, we should absolutely ensure that each `uid` in the `uof_officers` table should refer to a row in the `personnel` table of course. But the `uof_officers` should be a separate table. It's a many-to-one mapping between rows in `personnel` table and rows in `uof` table.